lnrpc/rpc.proto

```diff
@@ -641,6 +641,17 @@ service Lightning {
         };
     }
 
+    /** lncli: `getnodemetrics`
+    GetNodeMetrics returns node metrics calculated from the graph. Currently
+    the only supported metric is betweenness centrality of individual nodes.
+    */
+    rpc GetNodeMetrics (NodeMetricsRequest) returns (NodeMetricsResponse) {
+        option (google.api.http) = {
+            get: "/v1/graph"
```

Comment:
doesn't this need to be distinct rom `DescribeGraph`'s?

Reply:
sorry, copy pasta